lib/service_locator.rb

```diff
@@ -0,0 +1,19 @@
+require 'pry'
+class SerivceLocator
+  @settings = {}
+
+  def self.setup(&block)
+    block.call(self)
+  end
+
+  def self.method_missing(m, *args, &block)
```

Comment:
Unused method argument - `block`. If it's necessary, use `_` or `_block` as an argument name to indicate that it won't be used.